src/kagglehub/cache.py

```diff
@@ -4,7 +4,10 @@
 from typing import Optional
 
 from kagglehub.config import get_cache_folder
-from kagglehub.handle import ModelHandle, ResourceHandle
+from kagglehub.handle import DatasetHandle, ModelHandle, ResourceHandle
+
+DATASETS_CACHE_SUBFOLDER = "datasets"
+DATASETS_FILE_COMPLETION_MARKER_FOLDER = ".complete"
```

Comment:
nit: You can rename `MODELS_FILE_COMPLETION_MARKER_FOLDER` to `FILE_COMPLETION_MARKER_FOLDER` and use the same constant for both.